CHANGES.md

```diff
@@ -1,7 +1,17 @@
 Additions
 - New (experimental!) option `--as-json` for the HTML renderer that emits HTML
```

Comment:
Perhaps `unstable` is a better word to indicate that this might change in the future. (That's also what was decided to use upstream for OCaml 5 features)